tcmur_device.c

```diff
@@ -46,26 +46,15 @@ int __tcmu_reopen_dev(struct tcmu_device *dev, int retries)
 	int ret, attempt = 0;
 	bool needs_close = false;
 
+	assert(rdev->flags & TCMUR_DEV_FLAG_IN_RECOVERY);
```

Comment:
Shouldn't this be done with `rdev->rdev_lock` held (i.e. on line 50)?

Reply:
This is not that critical IMO. But I will fix it.